heron/api/src/java/com/twitter/heron/streamlet/StreamletNamePrefixType.java

```diff
@@ -0,0 +1,42 @@
+//  Copyright 2017 Twitter. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+package com.twitter.heron.streamlet;
+
+public enum StreamletNamePrefixType {
```

Comment:
This change is already included in an existing PR: https://github.com/twitter/heron/pull/2559/files#diff-59aa6c12ff571579c86e35a77cafe55eR95. I'd recommend leaving this change out of this PR to avoid conflict (I also would prefer for this enum to remain protected and restricted to the `StreamletImpl` class).